docs/source/tutorial/package_a_project.md

```diff
@@ -6,23 +6,23 @@ Kedro also has an advanced feature which supports packaging on a pipeline level
 
 ## Add documentation to your project
 
-Kedro uses the [Sphinx framework](https://www.sphinx-doc.org) to build any documentation that you create about your project.
+There are several documentation frameworks for Python projects, with [Sphinx](https://www.sphinx-doc.org) being among the most popular ones. You can leverage Sphinx to build the documentation of your Kedro project.
 
-To install a compatible version of Sphinx, run the following from the project root directory:
+To install Sphinx, run the following:
 
 ```bash
 pip install sphinx
 ```
 
 ### Set up the Sphinx project files
-Navigate to the `docs` directory of your Kedro project. Before Kedro version 0.19, the `docs/source` subdirectory was pre-populated with two Sphinx configuration files (`conf.py`, and `index.rst`) which the `kedro build-docs` command needed.
 
-From Kedro version 0.19 that command is deprecated, and those files will no longer exist. If you have used an earlier version of Kedro to create your project, and want to follow the following instructions to build the documentation without using `kedro build-docs`, you should delete the two files and the `source` subdirectory so that your project's `docs` directory is empty.
+.. warning::
```

Comment:
By adding this warning, all the rest of the instructions should be forward-compatible the moment `kedro new` does not create a `docs` directory anymore, as well as backwards compatible.